releasing/create-release.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# Copyright 2023 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# This script is called by Kustomize's release pipeline.
+# It needs jq (required for release note construction) and [GitHub CLI](https://cli.github.com/).
+#
+# To test it locally:
+#
+#   # Please install jq and GitHub CLI. (e.g. macOS)
+#   brew install jq gh
+#
+#   # Setup GitHub CLI
+#   gh auth login
+#
+#   # Run this script, where $TAG is the tag to "release" (e.g. kyaml/v0.13.4)
+#   ./releasing/create-release.sh $TAG
+#
+#   # Please remove Draft Release created by this script.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ -z "${1-}" ]]; then
+  echo "Usage: $0 TAG"
+  echo "  TAG: the tag to build or release, e.g. api/v1.2.3"
+  exit 1
+fi
+
+git_tag=$1
+echo "release tag: $git_tag"
+
+# Build the release binaries for every OS/arch combination.
+# It builds compressed artifacts on $release_dir.
+function build_kustomize_binary {
+  echo "build kustomize binaries"
+  version=$1
+
+  release_dir=$2
+  echo "build release artifacts to $release_dir"
+
+  mkdir -p "output"
+  # build date in ISO8601 format
+  build_date=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+  for os in linux darwin windows; do
+    for arch in amd64 arm64; do
```

Comment:
We have a couple other architectures for linux that we currently release for, linux s390x and linux ppc64le. You can see these on our releases, such as this one: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.0.1

Can we include s390x and ppc64le for linux here as well?